components/collective-page/hero/HeroAvatar.js

```diff
@@ -159,6 +172,67 @@ const HeroAvatar = ({ collective, isAdmin, intl, handleHeroMessage }) => {
             </div>
           )}
         </Dropzone>
+
+        <Container
+          onMouseEnter={() => {
+            setshowRemove(true);
+          }}
+          onMouseLeave={() => {
+            setshowRemove(false);
+          }}
+        >
+          {showRemove && (
+            <StyledButton
+              buttonSize="tiny"
+              minWidth={120}
+              css={{ position: 'absolute', marginTop: '-50px', marginLeft: '5px' }}
+              onClick={() => {
+                setshowModal(true);
+              }}
+            >
+              <Span ml={2} css={{ verticalAlign: 'center' }}>
+                <FormattedMessage id="HeroAvatar.Remove" defaultMessage="Remove logo" />
+              </Span>
+            </StyledButton>
+          )}
+        </Container>
+        {showModal && (
+          <ConfirmationModal
+            show={showModal}
+            width="100%"
+            maxWidth="570px"
+            onClose={() => {
+              setshowModal(false);
+            }}
+            header={<FormattedMessage id="heroavtar.remove" defaultMessage="Remove Logo?" />}
+            continueHandler={async () => {
+              setSubmitting(true); // Need this because `upload` is not a graphql function
+
+              try {
+                // console.log('imageURl is ',imgURL);
+                const imgURL = null;
+
+                // Update settings
+                await editImage({ variables: { id: collective.id, image: imgURL } });
+
+                // Reset
+                setUploadedImage(null);
+                setEditing(false);
+                setshowModal(false);
+              } finally {
+                setSubmitting(false);
+                handleHeroMessage(null);
+              }
```

Comment:
```suggestion
              setSubmitting(true); // Need this because `upload` is not a graphql function

              try {
                await editImage({ variables: { id: collective.id, image: null,  } });
                setshowModal(false);
              } finally {
                setSubmitting(false);
                handleHeroMessage(null);
              }
```